### PR TITLE
Added Installation to Docs

### PR DESCRIPTION
**Pull request by BolunDai0216**
# Description

Added installation instructions to the documentation website. 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `pre-commit run --all-files` (see `CONTRIBUTING.md` instructions to set it up)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: minigrid/minigrid_env.py [hunk 1]
```diff
@@ -196,6 +196,11 @@ def __str__(self):
 
         output = ""
 
+        # check if self.agent_pos & self.agent_dir is None
```
Comment (pseudo-rnd-thoughts) [inline minigrid/minigrid_env.py:199]: Hey, could we do this is a separate PR and do the full change to use the standard gymnasium implementation? 
I don't think we should hide this away in an unrelated PR

Comment (BolunDai0216) [reply minigrid/minigrid_env.py:199]: Yeah, that's probably better.
